### PR TITLE
Usability fixes

### PR DESCRIPTION
**Pull request by mariasereno-png**
## What it does, and why

**DESIGN-239 and DESIGN-218 fixes:**  
- Fix `Select` styling -- border-radius, colour, padding
- Fix `Small` element and props
- Add uppercase prop to `Typography` 
- Fix `Grid` and `Flex` default props
- Fix `Sidebar` filter badge alignment

**Jira cards:**
https://orchestrated.atlassian.net/browse/DESIGN-239
https://orchestrated.atlassian.net/browse/DESIGN-218

### CODE REVIEW

**Review by kchuah — APPROVED** [review]
lgtm

File: packages/orcs-design-system/lib/components/Flex/index.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ Flex.defaultProps = {
   flexDirection: "row",
   flexWrap: "nowrap",
   justifyContent: "flex-start",
-  alignItems: "center",
+  alignItems: "normal",
```
Comment (abottega) [inline packages/orcs-design-system/lib/components/Flex/index.js:78]: default for align-items is not normal, it's stretch

**Review by abottega — CHANGES_REQUESTED** [review]
Now on modal, clicking the overlay background closes the modal, but clicking the cross does not.

and default align-items for flex component needs to be fixed.

Other than that it looks good to me.

**Comment by mariasereno-png** [comment]
> Now on modal, clicking the overlay background closes the modal, but clicking the cross does not.
> 
> and default align-items for flex component needs to be fixed.
> 
> Other than that it looks good to me.

Re: default prop for align-items, I went by this: https://developer.mozilla.org/en-US/docs/Web/CSS/align-items

Re: the X bug, thanks for the catch -- I've updated the code in Modal.stories.js (basically, if you're using `useEffect` you have to add an event listener to the X as well, and no need to use onClick)

**Review by abottega — APPROVED** [review]

**Pull request merged**